### PR TITLE
feat: Add artisan db:show command

### PR DESCRIPTION
**Pull request by almas1992**
## 📑 Description

This PR introduces an `artisan db:show` command to display database information.

#### Screenshots
image

@coderabbitai summary

## ✅ Checks

- [x] Added test cases for my code

### CODE REVIEW

**Comment by almas1992** [comment]
Review Ready

File: database/console/show_command.go [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+	return "Display information about the given database"
+}
+
+// Extend The console command extend.
+func (r *ShowCommand) Extend() command.Extend {
+	return command.Extend{
+		Category: "db",
+		Flags: []command.Flag{
+			&command.StringFlag{
+				Name:  "database",
```
Comment (hwbrzzl) [inline database/console/show_command.go:57]:
```suggestion
				Name:  "connection",
				Aliases: []string{"c"},
```

File: database/console/show_command.go [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+				Usage: "The database connection",
+			},
+			&command.BoolFlag{
+				Name:  "views",
```
Comment (hwbrzzl) [inline database/console/show_command.go:61]:
```suggestion
				Name:  "views",
				Aliases: []string{"v"},
```

File: database/console/show_command.go [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+				name:                 "SQL Server",
+				versionQuery:         "SELECT SERVERPROPERTY('productversion') AS value;",
+				openConnectionsQuery: "SELECT COUNT(*) Value FROM sys.dm_exec_sessions WHERE status = 'running';",
+			},
+		}
+	)
+	name = string(r.schema.Orm().Query().Driver())
+	if driver, ok := drivers[r.schema.Orm().Query().Driver()]; ok {
+		name = driver.name
+		_ = r.schema.Orm().Query().Raw(driver.versionQuery).Scan(&result)
```
Comment (hwbrzzl) [inline database/console/show_command.go:132]: Don't ignore error.

File: database/console/show_command.go [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+		version = result.Value
+		if strings.Contains(version, "MariaDB") {
+			name = "MariaDB"
+		}
+		if len(driver.openConnectionsQuery) > 0 {
+			_ = r.schema.Orm().Query().Raw(driver.openConnectionsQuery).Scan(&result)
```
Comment (hwbrzzl) [inline database/console/show_command.go:138]: Ditto

File: database/console/show_command_test.go [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+	tests := []struct {
+		name     string
+		setup    func()
+		expected string
+	}{
+		{
+			name: "invalid argument",
+			setup: func() {
+				mockContext.EXPECT().Argument(0).Return("test")
```
Comment (hwbrzzl) [inline database/console/show_command_test.go:44]: And the same below.

```suggestion
				mockContext.EXPECT().Argument(0).Return("test").Once()
```

File: database/console/show_command_test.go [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
```
Comment (hwbrzzl) [inline database/console/show_command_test.go:45]: Some errors can't be thrown when using `mock.Anything`, so we should try to avoid using it.

**Review by hwbrzzl — COMMENTED** [review]
Amazing, only a few nitpicks.

File: database/console/show_command.go (hunk 1 above)
Comment (hwbrzzl) [reply database/console/show_command.go:57]: Just find it's `database` in Laravel, ignore this.

File: database/console/show_command_test.go [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+				mockSchema.EXPECT().GetTables().Return(nil, nil)
+				mockContext.EXPECT().OptionBool("views").Return(true)
+				mockSchema.EXPECT().GetViews().Return(nil, assert.AnError)
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		}, {
+			name: "success",
```
Comment (hwbrzzl) [inline database/console/show_command_test.go:92]: No expected in this case? We can use `Contains` to assert it if it's a bit complicated.

**Comment by almas1992** [comment]
Review Ready

File: database/console/show_command.go [hunk 8]
```diff
@@ -54,12 +54,14 @@ func (r *ShowCommand) Extend() command.Extend {
 		Category: "db",
 		Flags: []command.Flag{
 			&command.StringFlag{
-				Name:  "database",
-				Usage: "The database connection",
+				Name:    "connection",
```
Comment (hwbrzzl) [inline database/console/show_command.go:57]: Sorry, we can keep `database` here:
 
image

Comment (almas1992) [reply database/console/show_command.go:57]: Okay

**Comment by almas1992** [comment]
Review Ready

**Review by hwbrzzl — APPROVED** [review]
Perfect

**Pull request merged**